scripts/basic-workflow-tests.sh

```diff
@@ -133,13 +133,24 @@ test_generate() {
 test_train() {
     task Train the model
 
+    # Reduce testfile to 1 sample, generating from the new model is extreemly slow in CI
+    # generating only a single test should be enough as we're not verifying the output
+    TESTFILE=$(set +f ; ls -tr generated/test_*jsonl | tail -n 1)
+    mv $TESTFILE ${TESTFILE}_
+    head -n 1 ${TESTFILE}_ > $TESTFILE
+    rm ${TESTFILE}_
+
     # TODO Only cuda for now
-    TRAIN_ARGS="--device=cuda --4-bit-quant"
+    TRAIN_ARGS=()
+    TRAIN_ARGS+=("--device=cuda")
+    TRAIN_ARGS+=("--override-training-args")
+    TRAIN_ARGS+=("{\"bf16\":false, \"gradient_checkpointing\":true, \"gradient_accumulation_steps\":8}")
```

Comment:
this might be little prettier if you added single quotes and ditched the escaping...e.g.

```
TRAIN_ARGS+=('{
  "bf16": false,
  "gradient_checkpointing": true,
  "gradient_accumulation_steps": 8
}')
```